site/src/sphinx/setup-configuration.rst

```diff
@@ -268,3 +275,81 @@ Once you have an access to a ZooKeeper cluster, update the ``replication`` secti
 
   -  the minimum allowed age of log items before they are removed from ZooKeeper. If ``null`` the default
      value of '3600000 milliseconds' (1 hour) is used.
+
+Configuring TLS
+---------------
+Central Dogma supports TLS for its API and web pages. To enable TLS, a user may configure ``tls`` property
+in ``dogma.json`` as follows. Note that Central Dogma does not support Server Name Indication(SNI) extension
+at the moment.
+
+.. code-block:: json
+
+    {
+      "dataDir": "./data",
+      "ports": [
+        {
+          "localAddress": {
+            "host": "*",
+            "port": 36462
+          },
+          "protocol": "https"
+        }
+      ],
+      "tls": {
+        "keyCertChainFile": "./cert/centraldogma.crt",
+        "keyFile": "./cert/centraldogma.key",
+        "keyPassword": null
+      },
+      "numWorkers": null,
+      "maxNumConnections": null,
+      "requestTimeoutMillis": null,
+      "idleTimeoutMillis": null,
+      "maxFrameLength": null,
+      "numRepositoryWorkers": 16,
+      "cacheSpec": "maximumWeight=134217728,expireAfterAccess=5m",
+      "webAppEnabled": true,
+      "gracefulShutdownTimeout": {
+        "quietPeriodMillis": 1000,
+        "timeoutMillis": 10000
+      },
+      "replication": {
+        "method": "NONE"
+      },
+      "securityEnabled": false,
+      "mirroringEnabled": true,
+      "numMirroringThreads": null,
+      "maxNumFilesPerMirror": null,
+      "maxNumBytesPerMirror": null,
+      "accessLogFormat": "common"
+    }
+
+- ``tls``
+
+  - the configuration for TLS support. It will be applied to the port which is configured with ``https``
+    protocol. If ``null``, a self-signed certificate will be generated for ``https`` protocol.
+  - ``keyCertChainFile`` (string)
+
+    - the path to the certificate chain file.
+
+  - ``keyFile`` (string)
+
+    - the path to the private key file.
+
+  - ``keyPassword`` (string)
+
+    - the password of the private key file. Specify ``null`` if no password is set.
```

Comment:
How about adding:

    Note that `null` (no password) and `"null"` (password is 'null') are different.